core/src/main/scala/cats/mtl/Ask.scala

```diff
@@ -51,6 +51,11 @@ trait Ask[F[_], +E] extends Serializable {
   def ask[E2 >: E]: F[E2]
 
   def reader[A](f: E => A): F[A] = applicative.map(ask)(f)
+
+  def fromReader[A](ra: Reader[E, A]): F[A] = reader(ra.run)
+
+  def fromKleisli[A](ka: Kleisli[F, E, A])(implicit F: FlatMap[F]): F[A] =
+    ask.flatMap(ka.run(_))
```

Comment:
Andrew raised a great question about use-cases, which made me realize we can do even better 🙂 

For example suppose you have a `Ask[IO, Request[IO]]` and you want to interop with an http4s thing `Kleisli[IO, Request[IO], Response[IO]]`. Well, now you can!